tests/WebApiGateway.UnitTests/Infrastructure/Builders/Support.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace WebApiGateway.UnitTests.Infrastructure.Builders
+{
+    public static class Support
+    {
+        public static List<string> GenerateRoleIds(int rolesListSize)
```

Comment:
Rename method to GenerateGuidList(int size = 5)
Method should return IEnumerable<Guid>.